docs/view-style-props.md

```diff
@@ -289,6 +289,15 @@ If the rounded border is not visible, try applying `overflow: 'hidden'` as well.
 | ------ |
 | number |
 
+### `cursor` <div class="label ios">iOS</div>
+
+On iOS 17+, Setting this to `pointer` allows hover effects when a pointer (such as an Apple Pencil or trackpad on an iPad, or the users' gaze on an Apple Vision Pro) is over the view.
```

Comment:
I'm not a huge fan of listing specific devices, but for plain React Native (not React Native macOS, etc), this is where that prop applies and I think it's not obvious unless explicitly stated.